fiatconnect-api.md

```diff
@@ -2267,6 +2269,22 @@ The `ifsc` or Indian Financial System Code is an 11-digit alpha-numeric code tha
 The `accountNumber` field is unique between individuals and no two banks or account holders can have the same account number.
 Banks use different starting codes for their branches for differentiation. (In India, bank account numbers usually contain 7 to 21 digits.)
 
+#### 8.3.2.6. `PaymentCardAccount`
+
+`PaymentCardAccount` is an account schema representing transfer in type of transactions.
+
+```
+{ 
+    "name": "string",
+    "cardNumber": "string",
+    "expiry": "string",
+    "cvv": "string"
```

Comment:
Looks like this is missing fields that are required on fiat account schemas (accountName, institutionName, fiatAccountType)